src/masternode.cpp

```diff
@@ -517,7 +526,7 @@ bool GetMasternodeRanks()
         vecMasternodeScores.push_back(make_pair(value, &mn));
     }
 
-    sort(vecMasternodeScores.rbegin(), vecMasternodeScores.rend(), CompareLastPayRate());
+    sort(vecMasternodeScores.rbegin(), vecMasternodeScores.rend(), CompareLastPay());
```

Comment:
vecMasternodeScores is now vecFortunastakeScores i think?

Reply:
Yes it was committed in v3.2.